osx/ka-lite-remover.sh

```diff
@@ -0,0 +1,163 @@
+#!/bin/bash
+
+
+# Notes: 
+#    * This script must be run as root.
+#    * The files that will be removed, will be displayed on the console log.
+#    * The $SCRIPT_NAME env variables is used by the packages.
```

Comment:
@arceduardvincent This is confusing, can you re-phrase this by specifying where does SCRIPT_NAME get specified?  Is it done by the Packages tool itself or will we set it manually?